packages/datatrak-web/index.html

```diff
@@ -4,9 +4,13 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1"  />
     <meta name="theme-color" content="#000000" />
+    <!-- Open Graph for Facebook link preview -->
+    <meta property="og:image" content="https://datatrak.tupaia.org/social-preview.png" />
```

Comment:
In the spirit of accessibility, should we have an accompanying `og:image:alt`?

Reply:
Yes good point. I didn't know about that tag.